### ISSUE TITLE
Add LICENSE

### ISSUE THREAD

**Issue by andreiborisov** [malob/DarkMode.spoon#9]
I'd like to add this spoon as a dependency to my dotfiles repo (which is licensed under `MIT`). Would you be kind enough to add `LICENSE` file to the repo?

**Comment by malob** [comment]
Yup, I'll try and get that done by the end of the day. Thanks for reaching out and asking!
